src/components/pages/actor-page/actor-page.js

```diff
@@ -1,15 +1,34 @@
 
 import MovieList from '../../movie-list/movie-list';
 import ErrorBoundary from '../../error-boundary/error-boundary';
+import {img_300, notfound_300} from '../../../services/media-service';
 import { Container } from '@mui/material';
 import './actor-page.scss';
+import { SpinnerCircularFixed } from 'spinners-react';
 import avatar from './actor-avatar.jpg'
-import actor1 from './actor1.jpg';
-import actor2 from './actor2.jpg';
-import actor3 from './actor3.jpg';
-
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { nanoid } from '@reduxjs/toolkit';
+import { fetchPerson, fetchActorImages, fetchActorMovies } from '../../../redux/actorSlice';
 
 const ActorPage = () => {
+  const dispatch = useDispatch();
+  const lang = useSelector(state => state.appReducer.lang);
+  const {imagesStatus, moviesStatus} = useSelector(state => state.actorReducer)
+  const movies = useSelector(state => state.actorReducer.actorMovies);
+  const images = useSelector(state => state.actorReducer.actorImages);
```

Comment:
move selectors outside jsx
for all files

Reply:
Fixed!